ui/src/Room.tsx

```diff
@@ -122,6 +129,42 @@ export const Room = ({
 
     const controlVisible = showControl || open || showMore || hoverControl;
 
+    useHotkeys('s', () => (hostStream ? stopShare() : share()), [hostStream]);
```

Comment:
I don't think we need additional state like hostStream and clientStreams that is updated in a useEffect. state.hostStream should be used directly. Please do the same for clientStream.
```suggestion
    useHotkeys('s', () => (state.hostStream ? stopShare() : share()), [state.hostStream]);
```

Reply:
You are right. I've removed the unnecessary stage usage.